### PR TITLE
Warnings: various fixes

### PR DESCRIPTION
**Pull request by vovodroid**
Small fixes of various warnings, functionality shouldn't be affected.

### CODE REVIEW

File: src/slic3r/GUI/WebGuideDialog.cpp [hunk 1]
```diff
@@ -1139,8 +1139,8 @@ int GuideFrame::LoadProfile()
                 //cout << iter->path().string() << endl;
 
                 wxString strVendor = from_u8(iter->path().string()).BeforeLast('.');
-                strVendor          = strVendor.AfterLast( '\\');
-                strVendor          = strVendor.AfterLast('\/');
+                strVendor          = strVendor.AfterLast('\\');
+                strVendor          = strVendor.AfterLast('/');
```
Comment (SoftFever) [inline src/slic3r/GUI/WebGuideDialog.cpp:1143]: Is it suppose to be escaped?

File: src/slic3r/GUI/WebGuideDialog.cpp [hunk 2]
```diff
@@ -1656,7 +1656,7 @@ std::string GuideFrame::w2s(wxString sSrc)
 
 void GuideFrame::GetStardardFilePath(std::string &FilePath) {
     StrReplace(FilePath, "\\", w2s(wxString::Format("%c", boost::filesystem::path::preferred_separator)));
-    StrReplace(FilePath, "\/", w2s(wxString::Format("%c", boost::filesystem::path::preferred_separator)));
+    StrReplace(FilePath, "/" , w2s(wxString::Format("%c", boost::filesystem::path::preferred_separator)));
```
Comment (SoftFever) [inline src/slic3r/GUI/WebGuideDialog.cpp:1659]: Is it suppose to be escaped?
Seems the logic is reverted

File: src/slic3r/GUI/WebGuideDialog.cpp (hunk 1 above)
Comment (nevack) [reply src/slic3r/GUI/WebGuideDialog.cpp:1143]: I think regular slash `/` doesn't need ecsaping

Comment (vovodroid) [reply src/slic3r/GUI/WebGuideDialog.cpp:1143]: > regular slash `/` doesn't need ecsaping

Exactly.

File: src/slic3r/GUI/WebGuideDialog.cpp (hunk 2 above)
Comment (vovodroid) [reply src/slic3r/GUI/WebGuideDialog.cpp:1659]: As @nevack said above.

File: src/slic3r/GUI/Jobs/PrintJob.cpp [hunk 3]
```diff
@@ -446,7 +446,7 @@ void PrintJob::process(Ctl &ctl)
             std::string curr_job_id;
             json job_info_j;
             try {
-                job_info_j.parse(job_info);
+                (void)job_info_j.parse(job_info);
```
Comment (nevack) [inline src/slic3r/GUI/Jobs/PrintJob.cpp:449]: nit: Modern way to do this is `std::ignore`
See: https://en.cppreference.com/w/cpp/utility/tuple/ignore

Comment (vovodroid) [reply src/slic3r/GUI/Jobs/PrintJob.cpp]: Done, thanks.

File: src/slic3r/GUI/WebGuideDialog.cpp (hunk 1 above)
Comment (SoftFever) [reply src/slic3r/GUI/WebGuideDialog.cpp:1143]: cool
thanks

**Review by SoftFever — APPROVED** [review]
LGTM

**Pull request merged**